### DISCUSSION TITLE
Requirements for mandatory 2FA

### DISCUSSION THREAD

**Discussion by Hk1020** [community/community#65989] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

What are github’s requirements to make 2FA mandatory? I’d like to change my github content so I can disable the forced 2FA.

**Comment by supervisitor** [comment]
Nice approach! 😎 I open a ticket to get a answer to a similar topic (login without 2FA/mobile) from GitHub support directly. Part of the (marketing) answer was that they/GitHub/MS have general concerns about the security of other services... so they/GitHub/MS will only trust its own mechanism.

But in any case, if you want to generate a noticeable and countable feedback, you have to open a ticket and ask the support directly for the howto of a  non 2FA access. Or you have to leave to a similar service... the internet has more than one better competitor. 😇

**Comment by Hk1020** [comment]
> 
> Nice approach! 😎 I open a ticket to get a answer to a similar topic (login without 2FA/mobile) from GitHub support directly. Part of the (marketing) answer was that they/GitHub/MS have general concerns about the security of other services... so they/GitHub/MS will only trust its own mechanism.
> 
> But in any case, if you want to generate a noticeable and countable feedback, you have to open a ticket and ask the support directly for the howto of a non 2FA access. Or you have to leave to a similar service... the internet has more than one better competitor. 😇
> 
Actually, this wasn't meant as a sneaky way to get an answer. I am seriously interested to know what I need to remove from my github contributions so that the 2FA requirement is lifted. Their justification to require 2FA was something that I contributed code in a significant amount and that I therefore felt under the new guidelines. I just want to reverse this. I'd happily remove my two clones, none of which is relevant anymore. But before I do so I'd like to know if it would actually help. From your reply it seems only support can tell me. So, what are those guidelines?

**Reply by supervisitor** [reply]
... you need to open a ticket to the official support. Only they know the ruleset. And if you get an answer, please share the content with everyone here. I am also interested in the solution, but I have received only marketing babble ... they even doubt my access security to my mailbox! 😉 This is the reason why they don't want to stay with mail-passcode...
